instrumentation/opentelemetry-instrumentation-falcon/src/opentelemetry/instrumentation/falcon/__init__.py

```diff
@@ -83,19 +83,26 @@ class FalconInstrumentor(BaseInstrumentor):
 
     def _instrument(self, **kwargs):
         self._original_falcon_api = falcon.API
+        _InstrumentedFalconAPI._tracer_provider = kwargs.get("tracer_provider")
```

Comment:
Any reason not to store the `tracer` instead of `tracer_provider` here?

Reply:
I don't see any problem with storing tracer directly. Let me check and comeback.

Reply:
There is no special reason except the little lazy initialization of tracer. Either way is fine to me. What do you think?

Reply:
To be consistent, I think storing `tracer` would be nice.